tests/integration/multi_page/test_pages_layout.py

```diff
@@ -0,0 +1,136 @@
+import dash
+from dash import Dash, dcc, html
+
+
+def get_app(path1="/", path2="/layout2"):
+    app = Dash(__name__, use_pages=True)
+
+    # test for storing arbitrary keyword arguments: An `id` prop is defined for every page
+    # test for defining multiple pages within a single file: layout is passed directly to `register_page`
+    # in the following two modules:
+    dash.register_page(
+        "multi_layout1",
+        layout=html.Div("text for multi_layout1", id="text_multi_layout1"),
+        path=path1,
+        title="Supplied Title",
+        description="This is the supplied description",
+        name="Supplied name",
+        image="birds.jpeg",
+        id="multi_layout1",
+    )
+    dash.register_page(
+        "multi_layout2",
+        layout=html.Div("text for multi_layout2", id="text_multi_layout2"),
+        path=path2,
+        id="multi_layout2",
+    )
+
+    app.layout = html.Div(
+        [
+            html.Div(
+                [
+                    html.Div(
+                        dcc.Link(
+                            f"{page['name']} - {page['path']}",
+                            id=page["id"],
+                            href=page["path"],
+                        )
+                    )
+                    for page in dash.page_registry.values()
+                ]
+            ),
+            dash.page_container,
+        ]
+    )
+    return app
+
+
+def test_pala001_layout(dash_duo):
+
+    dash_duo.start_server(get_app())
+
+    # test layout and title for each page in `page_registry` with link navigation
+    for page in dash.page_registry.values():
+        dash_duo.find_element("#" + page["id"]).click()
+        dash_duo.wait_for_text_to_equal("#text_" + page["id"], "text for " + page["id"])
+        assert dash_duo.driver.title == page["title"], "check that page title updates"
+
+    # test redirects
+    dash_duo.wait_for_page(url="http://localhost:8050/v2")
+    dash_duo.wait_for_text_to_equal("#text_redirect", "text for redirect")
+    dash_duo.wait_for_page(url="http://localhost:8050/old-home-page")
+    dash_duo.wait_for_text_to_equal("#text_redirect", "text for redirect")
```

Comment:
Should the url have changed to `/redirect` in these cases? Can we test that?

Reply:
Added the test